types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+
+
+message Event {
+    google.protobuf.Timestamp timestamp = 1;
+    uint32 id = 2;
+    string name = 3;
+    Policies policies = 4;
+    Context context = 5;
+    // repeated EventData data = 6; TODO: event data is coming on a separate PR
+    optional Threat threat = 7;
+}
+
+message Policies {
+    repeated string matched = 1;
+}
+
+message Context {
+    optional Process process = 1;
+    optional Container container = 2;
+    optional K8S k8s = 3;
+}
+
+message Process  {
+    Executable executable = 1;
+    int32 pid = 2;
+    int32 namespaced_pid = 3;
+    uint32 entity_id = 4;
+    User real_user = 5;
+    Thread thread = 6;
+    optional Parent parent = 7;
+}
+
+message Parent  {
+    Executable executable = 1;
+    int32 pid = 2;
+    optional int32 namespaced_pid = 3;
+    uint32 entity_id = 4;
```

Comment:
I know that this is nit of nits :-) but do you mind placing entity_id between executable and pid?
Then all optional fields will be grouped together (if changing this please also do the same for the Process message)